File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/zig/F01.P2.zig
const std = @import("std");

// BEGIN PROBE F01.P2
var counter: i64 = 0;
const LIMIT: i64 = 100;
// END PROBE F01.P2

pub fn main() !void {
    counter += LIMIT;
    var threaded: std.Io.Threaded = .init_single_threaded;
    const io = threaded.io();
    var out_buf: [64]u8 = undefined;
    var out = std.Io.File.stdout().writer(io, &out_buf);
    try out.interface.print("{d}\n", .{counter});
    try out.interface.flush();
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F01.P2.java
class Main {
    // BEGIN PROBE F01.P2
    static long counter = 0;
    static final long LIMIT = 100;
    // END PROBE F01.P2

    public static void main(String[] args) {
        counter = counter + LIMIT;
        System.out.println(counter);
    }
}
